app/controllers/form_controller.rb

```diff
@@ -120,6 +121,11 @@ def is_referrer_check_answers?
     referrer.present? && CGI.parse(referrer.split("?")[-1]).present? && CGI.parse(referrer.split("?")[-1])["referrer"][0] == "check_answers"
   end
 
+  def interruprion_screen_referrer
+    referrer = request.headers["HTTP_REFERER"].presence || ""
+    return CGI.parse(referrer.split("?")[-1])["referrer"][0] if referrer.present? && CGI.parse(referrer.split("?")[-1]).present?
+  end
```

Comment:
This is very similar to the method at the top (`is_referrer_check_answers?`):
When we click change we get sent to `lettings-logs/<id>/<question_that_is_being_changed>?referrer=<soft_validation_path>` 
So at this point `request.headers["HTTP_REFERER"]` could for example be `lettings-logs/1/pregnant?referrer=no_females_pregnant_household_value_check`
And `CGI.parse(referrer.split("?")[-1])["referrer"][0]` would end up being `no_females_pregnant_household_value_check`

Reply:
Because the interruption_screen page is no longer routed to after the relevant soft validation has been fixed. We don't really want to route to it, but we want to right after some of the relevant questions have been updated according to design